fd_leak_comment.md

```diff
@@ -0,0 +1,85 @@
+Nothing actionable in this comment. Just a findings update for the curious and to help me gather my thoughts and remember the key facts.
```

Comment:
This probably shouldn't live as a top level md in the repo. Maybe put it as a comment near the relevant code.

Reply:
Sorry, I added this file to the PR by mistake. It was supposed to stay only in my local working branch.